src/main/java/org/myrobotlab/service/Tracking.java

```diff
@@ -329,64 +393,106 @@ public ServiceConfig load(ServiceConfig c) {
     }
     return config;
   }
-  
+
   public boolean isIdle() {
     return state == TrackingState.IDLE;
   }
 
-  public void startWithDefaults() {
-    if (pan == null) {
-      pan = String.format("%s.pan", getName());
-      Runtime.start(pan, "Servo");
-      attachPan(pan);
-    }
-    if (tilt == null) {
-      tilt = String.format("%s.tilt", getName());
-      Runtime.start(tilt, "Servo");
-      attachTilt(tilt);
-    }
-    if (pid == null) {
-      pid = String.format("%s.pid", getName());
-      Runtime.start(pid, "Pid");
-      attach(pid);
-    }
-
-    if (cv == null) {
-      cv = String.format("%s.cv", getName());
-      Runtime.start(cv, "OpenCV");
-      attach(cv);
-    }
+  static public LinkedHashMap<String, ServiceConfig> getDefault(String name) {

```

Comment:
add the @Override annotation to this method... you are overriding.

Reply:
Also this method signature isn't correct...  it's missing the Type...   (as per ServiceInterface.)